### ISSUE TITLE
Demo Polyhedron failed to build

### ISSUE THREAD

**Issue by beew** [CGAL/cgal#2223]
Hi, Tried to build CGAL 4.10 from source against qt-5.9.0  (and qt-5.9.1)  Make demos failed with
```
/home/bee/Downloads/CGAL-4.10/demo/Polyhedron/MainWindow.cpp: In member function ‘virtual void MainWindow::open(QString)’:
/home/bee/Downloads/CGAL-4.10/demo/Polyhedron/MainWindow.cpp:1027:34: error: no matching function for call to ‘QOpenGLContext::makeCurrent()’
   viewer->context()->makeCurrent();
                                  ^
In file included from /home/bee/opt/qt5/include/QtGui/qopenglfunctions_2_1.h:58:0,
                 from /home/bee/opt/qt5/include/QtGui/QOpenGLFunctions_2_1:1,
                 from /home/bee/Downloads/CGAL-4.10/demo/Polyhedron/../../include/CGAL/Three/Viewer_interface.h:30,
                 from /home/bee/Downloads/CGAL-4.10/demo/Polyhedron/../../include/CGAL/Three/TextRenderer.h:30,
                 from /home/bee/Downloads/CGAL-4.10/demo/Polyhedron/MainWindow.cpp:7:
/home/bee/opt/qt5/include/QtGui/qopenglcontext.h:172:10: note: candidate: bool QOpenGLContext::makeCurrent(QSurface*)
     bool makeCurrent(QSurface *surface);
          ^
/home/bee/opt/qt5/include/QtGui/qopenglcontext.h:172:10: note:   candidate expects 1 argument, 0 provided
demo/Polyhedron/CMakeFiles/polyhedron_demo.dir/build.make:134: recipe for target 'demo/Polyhedron/CMakeFiles/polyhedron_demo.dir/MainWindow.cpp.o' failed
make[3]: *** [demo/Polyhedron/CMakeFiles/polyhedron_demo.dir/MainWindow.cpp.o] Error 1
CMakeFiles/Makefile2:28944: recipe for target 'demo/Polyhedron/CMakeFiles/polyhedron_demo.dir/all' failed
make[2]: *** [demo/Polyhedron/CMakeFiles/polyhedron_demo.dir/all] Error 2
CMakeFiles/Makefile2:960: recipe for target 'CMakeFiles/demos.dir/rule' failed
make[1]: *** [CMakeFiles/demos.dir/rule] Error 2
Makefile:188: recipe for target 'demos' failed
make: *** [demos] Error 2
``` 

CGAL builds without problem.
OS is Ubuntu 16.04 64 bits.

**Comment by beew** [comment]
The culprit turns out to be libqglviewer-2.7.0. Compile went through with libqglviewer-2.6.4

**Comment by lrineau** [comment]
That is right. We found at that issue with recent daily tests, after our ArchLinux test platform had upgraded to libQGLViewer 2.7.0.

We have a patch in the pull-request #2213. For the moment, we do not plan to apply it to our CGAL-4.10 branch, but only to the `master` branch, that targets the future CGAL-4.11.

Do you think it should be backported to `releases/CGAL-4.10-branch`?

**Comment by lrineau** [comment]
The branch of CGAL-4.10.1 will be fixed by #2244.

**Comment by lrineau** [comment]
Fixed by #2244 in the future 4.10.1, and by #2213 for 4.11.
